### DISCUSSION TITLE
Space Lady - tryptech

### DISCUSSION THREAD

**Discussion by giscus** [tryptech/tryptech.github.io#1] [General]
# Space Lady - tryptech

Devlog for the development timeline of ZSSv3 for P+ 3.0

https://tryp.tech/post/space-lady

**Comment by Gonderage** [comment]
i saw the butt having uv sphere topology and i immediately knew that was gonna cause weighting problems lmao ive been there. blocky butts are the way to go nowadays, i suppose, since i see it done with Ultimate's models,

**Comment by davidvkimball** [comment]
Thanks for documenting all of this, extremely cool to see the progress. Cheers.
